.pre-commit-config.yaml

```diff
@@ -3,4 +3,8 @@ repos:
     rev: 23.3.0
     hooks:
     - id: black
-      language_version: python3.9
\ No newline at end of file
+      language_version: python3.9
```

Comment:
We still have Python 3.9? I mean, it doesn't matter that much, but in general we should be using 3.11

Reply:
Changed to 3.11